pkgs/top-level/aliases.nix

```
@@ -1527,6 +1527,8 @@ mapAliases ({
   transmission-qt = lib.warn (transmission3Warning {suffix = "-qt";}) transmission_3-qt; # Added 2024-06-10
   treefmt = treefmt2; # 2024-06-28
   libtransmission = lib.warn (transmission3Warning {prefix = "lib";}) libtransmission_3; # Added 2024-06-10
+  tracker = tinysparql; # Added 2024-09-05
+  tracker-miners = localsearch; # Added 2024-09-05
```

Shouldn't this use `lib.warn`?

```suggestion
  tracker = lib.warn "tracker has been renamed to tinysparql" tinysparql; # Added 2024-09-05
  tracker-miners = lib.warn "tracker-miners has been renamed to localsearch" localsearch; # Added 2024-09-05
```